app/lib/exceptionalTimesMapper.js

```diff
@@ -5,12 +5,14 @@ const futureDays = 14;
 
 function getDates(times) {
   const filteredTimes = {};
+  const dates = Object.keys(times);
 
-  Object.keys(times).forEach((exceptionalTime) => {
-    if (openingTimesUtils.isDateInWindow(exceptionalTime, timeUtils.getToday(), futureDays)) {
-      filteredTimes[exceptionalTime] = times[exceptionalTime];
+  for (let i = 0; i < dates.length; i++) {
+    const date = dates[i];
```

Comment:
Could `for (date of dates) { ` be used instead for the array iterations throughout the changes?

Reply:
I've had a think about the changes and decided that until there is a performance issue this change isn't a good one to make. Closing PR.